lib/active_merchant/billing/gateways/litle.rb

```diff
@@ -320,6 +320,7 @@ def add_auth_purchase_params(doc, money, payment_method, options)
         add_merchant_data(doc, options)
         add_debt_repayment(doc, options)
         add_stored_credential_params(doc, options)
+        add_fraud_filter_override(doc, options)
```

Comment:
I added it to `auth_purchase_params` but it may need to be added to `echeck_purchase_params` as well.

Reply:
Hmm...it's a little bit ambiguous from the Vantiv/WP docs, but what I see is that they call `fraudFilterOverride` a child element of either `authorization` or `sale`. Perhaps it depends on whether they consider `echeckSale` to be an entirely different thing than a `sale` or just a specific type of `sale`?

I think I would probably leave it as you have it here.